beacon_node/beacon_chain/src/block_verification.rs

```diff
@@ -652,7 +659,7 @@ pub fn signature_verify_chain_segment<T: BeaconChainTypes>(
     }
 
     if signature_verifier.verify().is_err() {
-        return Err(BlockError::InvalidSignature);
+        return Err(BlockError::InvalidSignature(InvalidSignature::Unknown));
```

Comment:
Why isn't this BlockBodySignatures

Reply:
This signature set includes all signatures (block body + proposer signature) of multiple blocks. This codepath is used only for sync by range, so it's not useful to figure out if it's the proposer signature. So I set here to unknown to avoid having to re-verify all proposer signatures for no benefit.

Reply:
Okay that makes sense